### PR TITLE
Publish non-repudiation-core to Artifactory as well

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
changelog_begin
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
:trollface: Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ci/copy-unix-release-artifacts.sh [hunk 1]
```diff
@@ -50,12 +50,26 @@ if [[ "$NAME" == "linux" ]]; then
...
+    cp bazel-bin/runtime-components/non-repudiation-core/libnon-repudiation-core.jar $OUTPUT_DIR/artifactory/$NON_REPUDIATION_CORE_JAR
+    cp bazel-bin/runtime-components/non-repudiation-core/non-repudiation-core_pom.xml $OUTPUT_DIR/artifactory/$NON_REPUDIATION_CORE_POM
+    cp bazel-bin/runtime-components/non-repudiation-core/libnon-repudiation-core-src.jar $OUTPUT_DIR/artifactory/$NON_REPUDIATION_CORE_SRC
+    cp bazel-bin/runtime-components/non-repudiation-core/non-repudiation-core_javadoc.jar $OUTPUT_DIR/artifactory/$NON_REPUDIATION_CORE_DOC
+
+
     NON_REPUDIATION_CLIENT_JAR=non-repudiation-client-$RELEASE_TAG.jar
     NON_REPUDIATION_CLIENT_POM=non-repudiation-client-$RELEASE_TAG.pom
     NON_REPUDIATION_CLIENT_SRC=non-repudiation-client-$RELEASE_TAG-sources.jar
-    NON_REPUDIATION_CLIENT_DOC=non-repudiation-cleint-$RELEASE_TAG-javadoc.jar
```
Comment (stefanobaghino-da) [inline ci/copy-unix-release-artifacts.sh:56]: Yet another 🤦🏻 prevented.

**Review by cocreature — APPROVED** [review]

File: ci/copy-unix-release-artifacts.sh [hunk 2]
```diff
@@ -50,12 +50,26 @@ if [[ "$NAME" == "linux" ]]; then
...
+    NON_REPUDIATION_CLIENT_DOC=non-repudiation-client-$RELEASE_TAG-javadoc.jar
     bazel build \
-          //runtime-components/non-repudiation-client \
+          //runtime-components/non-repudiation-client/... \
```
[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply ci/copy-unix-release-artifacts.sh:72]: Not really, but without this the other library didn't produce the POM for some reason and it started to make me feel shaky.

[the remark this answers is not in the document]
Comment (cocreature) [reply ci/copy-unix-release-artifacts.sh:72]: makes sense :+1:

File: ci/copy-unix-release-artifacts.sh [hunk 3]
```diff
@@ -50,12 +50,26 @@ if [[ "$NAME" == "linux" ]]; then
     bazel build //runtime-components/non-repudiation-app:non-repudiation-app_deploy.jar
     cp bazel-bin/runtime-components/non-repudiation-app/non-repudiation-app_deploy.jar $OUTPUT_DIR/artifactory/$NON_REPUDIATION
 
+    NON_REPUDIATION_CORE_JAR=non-repudiation-core-$RELEASE_TAG.jar
```
[the remark this answers is not in the document]
Comment (stefanobaghino-da) [reply ci/copy-unix-release-artifacts.sh:53]: Tracked by https://github.com/digital-asset/daml/issues/9306

**Pull request merged**